doc/Notebook_review_guide.md

```diff
@@ -0,0 +1,120 @@
+Login to ESIP-QHUB and start a server instance.
+Click on the *Git* dropdown from the top navigation bar and select *Clone a repo*. Copy and paste the SSH url from the *hytest-org* repo (or forked repo) into the dialog box and click *Clone*
+Informal Review
+
+Navigate to the hytest/dataset_preprocessing/tutorials/CONUS404 folder and open the CONUS404_DRB_Data_Prep.ipynb, selecting the "users-pangeo" kernel when asked.
+Based on the review type Andrew has asked you for and any additional you wish to do, use the questions as guidelines for any changes and feedback that you make:
+Feedback should be in the format of "[first name] comment/edited text/edited code"
+**Code review**
+Is the code readable? Do variable names make sense? Are all of the packages and tools used? Are functions adequately explained, including docstrings? Is there unnecessary duplication of code? Is there a more effective way to accomplish what the code does? What needs further explanation?
+
+**Domain review**
+Is the correct terminology used? Are variables consistent between datasets, example: are mean temp calculated correctly? Does the notebook take into account what domain experts would know? Are units correct? What needs further explanation?
+
+**Learner review**
+Do the learning objectives of the notebook seem clear? Are the sections organized in a way that makes sense? Where are sections that need more instruction or background? Are there sections that are repetitve? to What needs further explanation?
+
+Additionally, feedback on grammar or other thoughts are also welcome
+After adding all comments and suggested changes to the notebook, open a pull request in the [hytest-org repo](https://github.com/hytest-org/hytest) with the target branch of "arl-conus404_nb_review" and assign "alaws-USGS" as the reviewer. For those who forked the repo, follow the directions for [contributing to projects](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) that was used earlier.
+Please submit all feedback and changes via pull request by January 20th, 2023. Thank you for your help and if you have any questions, please feel free to contact Andrew.
+
+## Notebook Merge Conflicts
+
+When attempting to merge the branches, there is a high likelihood of merge conflicts. This is due to the structure of Jupyter Notebooks, which are an open document format based on JSON. What is seen by the person running the notebook cells are a rendered visualization so changes to a cell may cause issues in the underlying  JSON. Both GitHub and GitLab have write ups for handling merge conflicts. Here is the link to [Github's guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line) and here is [GitLab's guide](https://docs.gitlab.com/ee/user/project/merge_requests/conflicts.html).
```

Comment:
I'd invert the first part of this -- 
Emphasize that notebooks are composed largely of text we don't see and can't edit directly.